ansible/roles/service/hippo_authoring/files/hippo_authoring.yml

```diff
@@ -0,0 +1,280 @@
+      InstanceMonitoring: !Ref InstanceMonitoring
+      InstanceType: !Ref InstanceType
+      KeyName: !Ref KeyName
+      SecurityGroups:
+        - Ref: SG
+      UserData:
+        Fn::Base64: !Sub |
+          #!/bin/bash
+          export HOME=/root
+          . /bootstrap/bootstrap.inc.sh
+          set_global 'LaunchConfig' 'ASGroup' '${AWS::Region}' '${AWS::StackName}'
+          cfnInit
+          cd /bootstrap && make ansible_configure_local ROLE=hippo_authoring \
+            || die "configure playbook failed"
+          log "Instance bootstrapped"
+          signal 0
+          cleanup
+
+  ASGroup:
+    Type: AWS::AutoScaling::AutoScalingGroup
+    Properties:
+      DesiredCapacity: !Ref InstanceCount
+      LaunchConfigurationName: !Ref LaunchConfig
+      LoadBalancerNames:
+        - !Ref LoadBalancer
+      MaxSize: 6
+      MinSize: !Ref InstanceCount
+      Tags:
+        - Key: Name
+          Value: !Sub "${Environment}-hippo-authoring"
+          PropagateAtLaunch: true
+      VPCZoneIdentifier:
+        Fn::Split:
+          - ","
+          - Fn::ImportValue: !Sub "${Environment}-vpc::SubnetsPrivate"
+    CreationPolicy:
+      ResourceSignal:
+        Count: 0
+        Timeout: PT10M
+    UpdatePolicy:
+      AutoScalingRollingUpdate:
+        MinInstancesInService: 1
+        MaxBatchSize: 1
+        PauseTime: PT10M
+        WaitOnResourceSignals: true
+
+  ##
+  # Loadbalancing
+  #
+  LoadBalancerSg:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: Enable acces from VPC
+      VpcId:
+        Fn::ImportValue: !Sub "${Environment}-vpc::VpcId"
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          FromPort: 80
+          ToPort: 80
+          CidrIp: 156.67.242.53/32
+        - IpProtocol: tcp
+          FromPort: 443
+          ToPort: 443
+          CidrIp: 156.67.242.53/32
+      Tags:
+        - Key: Name
+          Value: !Sub "${Environment}-hippo-authoring"
+
+  LoadBalancer:
+    Type: AWS::ElasticLoadBalancing::LoadBalancer
+    Properties:
+      ConnectionDrainingPolicy:
+        Enabled: true
+        Timeout: 30
+      CrossZone: true
+      # HealthCheck:
+      #   Target: HTTP:8080/health-check
+      #   HealthyThreshold: 2
+      #   UnhealthyThreshold: 3
+      #   Interval: 10
+      #   Timeout: 2
+      Listeners:
+        - LoadBalancerPort: 80
+          Protocol: HTTP
+          InstancePort: 8080
+          InstanceProtocol: HTTP
+        # - LoadBalancerPort: 443
+        #   Protocol: HTTPS
+        #   SSLCertificateId: !Ref SSLCertificateId
+        #   InstancePort: 8080
+        #   InstanceProtocol: HTTP
+      Scheme: internet-facing
+      SecurityGroups:
+        - !Ref LoadBalancerSg
+      Subnets:
+        Fn::Split:
+          - ","
+          - Fn::ImportValue: !Sub "${Environment}-vpc::SubnetsPublic"
+      Tags:
+        - Key: Name
+          Value: !Sub "${Environment}-hippo-authoring"
+
+  ELBDNSRecordPublic:
```

Comment:
Maybe it's just me, but I find this name pretty hard to parse! Would you object to using C#-style aggressive Camel Casing: ElbDnsRecordPublic? Or, even easier to read: ElbDnsPublicRecord?

Reply:
It should have been `ElbDnsPublicRecord` :) just like everything else in the CF tpl